client/templates/teach/teach.js

```diff
@@ -30,7 +30,8 @@ Template.teach.events({
             author: template.find('#authorName').value, // string
             keywords: template.find('#courseKeywords').value.split(','), // split keywords to array
             published: template.find('#coursePublished').value, // string
-            about: template.find('#aboutText').value // Get the about text
+            about: template.find('#aboutText').value, // Get the about text
+            owner: currentUsername
```

Comment:
We need to plan this feature a bit better, and include it in a separate pull request. This solution only allows for one 'owner' string. Specifically, what if courses can have multiple 'owners' or 'collaborators?